test/rfc7540_SUITE.erl

```diff
@@ -2988,6 +2991,83 @@ client_settings_disable_push(Config) ->
 %   cause flow control or header compression state to become
 %   unsynchronized.
 %
+
+alpn_graceful_shutdown_client_stays(Config) ->
```

Comment:
Test has nothing to do with `alpn_` so remove this part.

Reply:
The alpn prefix makes it run with TLS because of logic in groups/0. I thought it's more realistic to test it with TLS, but can remove it.

Reply:
This test suite only uses TLS where relevant with regard to the RFC. Other test suites use TLS for just about everything you can do with Cowboy.